<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FinalizeOptionsWidget</class>
 <widget class="QWidget" name="FinalizeOptionsWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>250</width>
    <height>200</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Finalize</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <widget class="QGroupBox" name="colorModeGroup">
     <property name="title">
      <string>Color Mode</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <item>
       <widget class="QComboBox" name="colorModeCombo"/>
      </item>
      <item>
       <widget class="QLabel" name="statusLabel">
        <property name="text">
         <string>Not yet processed</string>
        </property>
        <property name="alignment">
         <set>Qt::AlignCenter</set>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="thresholdLayout">
        <item>
         <widget class="QLabel" name="thresholdLabel">
          <property name="text">
           <string>Midtone Threshold:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSpinBox" name="thresholdSpinBox">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>20</number>
          </property>
          <property name="value">
           <number>8</number>
          </property>
          <property name="suffix">
           <string>%</string>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QPushButton" name="applyToBtn">
        <property name="text">
         <string>Apply To ...</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="cacheGroup">
     <property name="title">
      <string>Detection Cache</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_3">
      <item>
       <widget class="QPushButton" name="clearCacheBtn">
        <property name="text">
         <string>Clear This Page</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QPushButton" name="clearAllCacheBtn">
        <property name="text">
         <string>Clear All Pages</string>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <widget class="QGroupBox" name="outputImagesGroup">
     <property name="title">
      <string>Output Images</string>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_4">
      <item>
       <widget class="QCheckBox" name="preserveOutputCB">
        <property name="text">
         <string>Preserve output images</string>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="outputFolderLayout">
        <item>
         <widget class="QPushButton" name="chooseOutputFolderBtn">
          <property name="text">
           <string>Choose Output Folder...</string>
          </property>
          <property name="enabled">
           <bool>false</bool>
          </property>
         </widget>
        </item>
       </layout>
      </item>
      <item>
       <widget class="QLabel" name="outputPathLabel">
        <property name="text">
         <string>Using temporary folder</string>
        </property>
        <property name="wordWrap">
         <bool>true</bool>
        </property>
       </widget>
      </item>
      <item>
       <layout class="QHBoxLayout" name="formatLayout">
        <item>
         <widget class="QLabel" name="formatLabel">
          <property name="text">
           <string>Format:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="formatCombo"/>
        </item>
        <item>
         <spacer name="formatSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="compressionLayout">
        <item>
         <widget class="QLabel" name="compressionLabel">
          <property name="text">
           <string>Compression:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QComboBox" name="compressionCombo"/>
        </item>
        <item>
         <spacer name="compressionSpacer">
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
          <property name="sizeHint" stdset="0">
           <size>
            <width>40</width>
            <height>20</height>
           </size>
          </property>
         </spacer>
        </item>
       </layout>
      </item>
      <item>
       <layout class="QHBoxLayout" name="jpegQualityLayout">
        <item>
         <widget class="QLabel" name="jpegQualityLabel">
          <property name="text">
           <string>Quality:</string>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QSlider" name="jpegQualitySlider">
          <property name="minimum">
           <number>1</number>
          </property>
          <property name="maximum">
           <number>100</number>
          </property>
          <property name="value">
           <number>90</number>
          </property>
          <property name="orientation">
           <enum>Qt::Horizontal</enum>
          </property>
         </widget>
        </item>
        <item>
         <widget class="QLabel" name="jpegQualityValueLabel">
          <property name="text">
           <string>90</string>
          </property>
          <property name="minimumSize">
           <size>
            <width>25</width>
            <height>0</height>
           </size>
          </property>
         </widget>
        </item>
       </layout>
      </item>
     </layout>
    </widget>
   </item>
   <item>
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
